.github/workflows/all_solutions.yml

```diff
@@ -636,7 +636,7 @@ jobs:
     runs-on: windows-2022
     strategy:
       matrix:
-        namespace: [ CosmosDB, Couchbase, MongoDB, Msmq, MsSql, MySql, NServiceBus, NServiceBus5, Oracle, Postgres, RabbitMq, Redis ]
+        namespace: [ CosmosDB, Couchbase, MongoDB, Msmq, MsSql, MySql, NServiceBus, NServiceBus5, Oracle, Postgres, RabbitMq, Redis, Elasticsearch ]
```

Comment:
OCD Alert: Could we maintain alphabetical order here (and in the other workflow)?